mage/main_test.go

```diff
@@ -1235,7 +1236,7 @@ func TestCompiledEnvironmentVars(t *testing.T) {
 	if err == nil {
 		t.Fatalf("expected an error because of timeout")
 	}
-	got = stdout.String()
+	got = stderr.String()
```

Comment:
These changes were required after changing the timeout handling code in `template.go`, now that it returns an error the message is sent to stderr.